.circleci/config.yml

```diff
@@ -97,6 +95,26 @@ jobs:
               done
               exit $ret
             fi
+  build-push-rust-code:
```

Comment:
nit: probably indicate that it's for docker images: `build-push-docker-rust-code`?